### ISSUE TITLE
Provide immutable source archives

### ISSUE THREAD

**Issue by jurajlutter** [hatoo/oha#145]
Hi,

in order to make life for us (package maintainers) a bit easier, I wonder if it's possible to release also properly rolled, immutable, source archives along with binary files.

The way it is done now relies on github-created archives that can, under some circumstances, change, causing checksums to also change (among other things).

What do you think about it?
Thanks.

**Comment by hatoo** [comment]
I think it's good to provide immutable source archives.
But I don't know how to do.
Is it can be done by modifying GitHub Action settings?
PR is more than welcome.

**Comment by jurajlutter** [comment]
Yes, you probably need to create the proper archive and release it as an `asset` similarly to what you already do with binary files.
